spec/controllers/feedbacks_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe FeedbacksController, type: :controller do
```

Comment:
Haha thank you for including tests we don't do nearly enough of these. However one thing we *do* usually do is to add basic postman tests that validate each endpoint. Those live in the postman directory and are used by TravisCI so please include some for your new routes.

Reply:
it was generated by Rails I can't get any credit for these lol